test/utest/src/cuter_cerl_tests.erl

```diff
@@ -79,10 +79,45 @@ construct_and_access_kfun_test() ->
 
 -spec construct_and_access_kmodule_test() -> any.
 construct_and_access_kmodule_test() ->
-  Code = cerl:c_fun([cerl:c_nil()], cerl:c_nil()),
+  Code = {c_fun,
+	  [{function,{f,1}},4],
+	  [{c_var,[4],0}],
+	  {c_var,[{function,{f,1}},4],0}},
   Kfun = cuter_cerl:kfun(Code, true),
-  Mfa = {some_module, some_fun, 1},
-  Kmodule = cuter_cerl:kmodule(some_module, nil, nil, [{Mfa, Kfun}]),
+  Mfa = {some_module, f, 1},
+  AST = {c_module,[],
```

Comment:
IMO, it's a very bad idea to be hard-coding the structure of the AST rather than building it using the `cerl` API.
For example, one can use `cerl:abstract(some_module)` to create the `{c_literal,[],some_module}` term, use `cerl:ann_abstract(file, [1])` to create `{c_literal,file,[1]}`, a call to `cerl:c_fun/2` for the `{c_fun, ...}` tuple, etc.

This way, this code is both easier to understand and protected from changes in the cerl representation.